### PR TITLE
Added a series of product tour pop-up in the dashboard page using Intro.js-react

### PR DESCRIPTION
**Pull request by hiroshinishio**
## Problem

There was no product tour. Therefore, even when new users logged in, they often did not make it to the user setup process. The completion rate of user setup after account creation was probably less than 20%. This was a problem.

## Solution

Product tours were implemented. Also called onboarding. Also called tech touch.
The implementation used a library called Intro.js and a React wrapper called Intro.js-react.

## Evidence

For desktop screens;

Pressing the "Start Tutorial" button appears to start the tour.

Dashboard - WorkStats - Google Chrome - 9 August 2022

Once you see the tour, reloading will not start the tour again.

Dashboard - WorkStats - Google Chrome - 9 August 2022 (1)

### Environment Variables Setting

Do you require registration for the following console screens beside the source code? If so, please attach evidence of registration to each of the console screens below.

- GitHub Actions: No
- Vercel Hosting: No
- Firebase Console: No
- Google Cloud Platform: No
- Asana Developer Console: No
- Slack Developer Console: No

## Performance

No

## Help Pages

No, This is a help in itself.

## Caveats

Notes.
Inevitably, due to a library glitch, we had to make it so that the user cannot leave the first time the popup is displayed, even if the user clicks on the escape button or outside of the popup.

### Documents

- [Intro.js Official Documents](https://introjs.com/)
- [Intro.js-react GitHub READ.me](https://github.com/HiDeoo/intro.js-react#introjs-options)

### Questions

- [ReferenceError: Element is not defined (Object.<anonymous>) in Intro.js-react, React, Next.js and Tailwind CSS](https://stackoverflow.com/questions/73267875/referenceerror-element-is-not-defined-object-anonymous-in-intro-js-react-r)
- [In Intro.js and Intro.js-react, the first and second steps are positioned not properly](https://stackoverflow.com/questions/73277077/in-intro-js-and-intro-js-react-the-first-and-second-steps-are-positioned-not-pr)
- [In "Intro.js" and "Intro.js-react", the first step appears twice](https://stackoverflow.com/questions/73278824/in-intro-js-and-intro-js-react-the-first-step-appears-twice)

### Blog

- https://zenn.dev/syu/articles/3c4aa813b57b8c

### CODE REVIEW

File: components/common/GearIcon.tsx [hunk 1]
```diff
@@ -8,13 +8,14 @@ interface GearIconType {
   mb: number;
   href: string;
   alt: string;
+  id?: string;
 }
 
-const GearIconLink = ({ mt, mb, href, alt }: GearIconType) => {
+const GearIconLink = ({ mt, mb, href, alt, id }: GearIconType) => {
```
Comment (peeyalk) [inline components/common/GearIcon.tsx:14]: when the number of props increases it is better not to destructure the object.

such as 
```ts
interface ObjectInterface {
    a: number,
    b: string
}
```

```ts 
const component = (object: ObjectInterface) => {
     // access the Object as such
     object.a...

}
```

Comment (peeyalk) [inline components/common/GearIcon.tsx:14]:
```ts
const GearIconLink = (gearIconType: GearIconType) => {
... 
      <a id={gearIconType.id} target='_blank' className={`mt-${mt} mb-${mb} ml-2`}>
      
```

Comment (hiroshinishio) [reply components/common/GearIcon.tsx:14]: @peeyalk Thanks for the comment, I've added it to the Asana task and will address it in another pull request!

Comment (hiroshinishio) [reply components/common/GearIcon.tsx:14]: @peeyalk Thanks for the comment, I've added it to the Asana task and will address it in another pull request!

**Review by peeyalk — APPROVED** [review]

**Pull request merged**